openmetadata-ui/src/main/resources/ui/src/utils/PermissionsUtils.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ *  Copyright 2021 Collate
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import { EntityType } from '../enums/entity.enum';
+import {
+  Access,
+  ResourcePermission,
+} from '../generated/entity/policies/accessControl/resourcePermission';
+import { Operation } from '../generated/entity/policies/policy';
+
+export const hasPemission = (
+  operation: Operation,
+  entityType: EntityType,
+  permissions: ResourcePermission[]
```

Comment:
Why are we passing permissions as param can't we access it directly from AppState?